cli/bash/VERSION

```diff
@@ -0,0 +1 @@
+2.0.1
```

Comment:
Out of curiosity, what is this used for?

Reply:
Version of the program.